### PR TITLE
thumbnail redesign

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
https://jira.sng.sk/browse/NSNG-82

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
                 full: '100%',
             },
             strokeWidth: {
-                2: '2px',
+                3: '3px',
```
Comment (igor-kamil) [inline tailwind.config.js:9]: please keep the `stroke-2` ;) 
```suggestion
                2: '2px',
                3: '3px',
```

**Review by igor-kamil — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (rastislav-chynoransky) [reply tailwind.config.js:9]: This is part of tailwind defaults: https://tailwindcss.com/docs/stroke-width

Comment (rastislav-chynoransky) [reply tailwind.config.js:9]: It's defined in the viewBox units, I see. Then I'd move this part from `extend` to overrides.

Comment (igor-kamil) [reply tailwind.config.js:9]: Ok. If it´s tested and works, lets keep it like this

**Review by igor-kamil — APPROVED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (rastislav-chynoransky) [reply tailwind.config.js]: Ok, I've changed it to unitless value.

**Pull request merged**